File: Unidad_2/02_Lambda_con_paramatros/01_Lambda_ejemplo/LambdaTest.java
public class LambdaTest{
    public static void main(String[] args) {
        //Expreción lambda => representa un objeto de una interfaz funcional
        Operaciones op = (num1, num2) -> System.out.println(num1 + num2); //implementacion del metodo "saludar()"
                                                                // de la Interfaz Funcional<
        op.imprimeSuma(5, 10);
    }
}

File: 2 _oportunidad/_04_ejercicios/lambda_con_parametros_2/LambdaTest.py
class LambdaTest:
    @staticmethod
    def main(args):
        op = lambda num1, num2 : print("El resultado es: "+(num1+num2))

        objeto = LambdaTest()
        objeto.miMetodo(op, 10, 10)

    def miMetodo(self, op, num1, num2):
        op.imprimeSuma(num1, num2)

